pymc_marketing/clv/models/beta_geo.py

```diff
@@ -344,3 +346,67 @@ def expected_num_purchases_new_customer(
         return (left_term * right_term).transpose(
             "chain", "draw", "t", missing_dims="ignore"
         )
+
+    def sample_population_distributions(
```

Comment:
Maybe keep the name of the methods more similar to those in the gamma-gamma? Something like `distribution_new_customer_dropout` and alike. Internally they can use this method which should be pretty cheap anyway.

I think that will make them more discoverable

Reply:
Do you think this method could be moved into `BaseModel` and standardized across all models?

Reply:
Good thinking. 
I view this as the closest solution

https://github.com/pymc-labs/pymc-marketing/blob/d4feeeafad6f0299b115dac3582ad6d98bd8241f/pymc_marketing/clv/models/shifted_beta_geo.py#L158-L201